undertow/src/main/java/org/wildfly/extension/undertow/security/AccountImpl.java

```diff
@@ -77,7 +77,7 @@ public AccountImpl(final Principal principal, Set<String> roles, final Object cr
 
     void setRoles(final Set<String> roles) {
```

Comment:
Having a quick look I don't see anywhere this is used - can it be removed instead?

Reply:
Thanks for your feedback. I have removed the method as you suggested.